stl/inc/algorithm

```diff
@@ -386,6 +386,21 @@ namespace ranges {
     inline constexpr _For_each_n_fn for_each_n{_Not_quite_object::_Construct_tag{}};
 
     // VARIABLE ranges::find
+    // concept-constrained for strict enforcement as it is used by several algorithms.
+    // clang-format off

```

Comment:
Nitpick: Comments should both begin with a capital and end in punctuation or neither begin with a capital nor end in punctuation. Also, it seems to me that the `concept-constrained` comment should bind tighter than `clang-format off`:
```suggestion
    // clang-format off
    // concept-constrained for strict enforcement as it is used by several algorithms
```